### PR TITLE
update section

### PR DESCRIPTION
**Pull request by EgleJuske**
- responsive design
- no header design
- icon gradient color made with -webkit- (couldn't find how to do it with regular css)

### CODE REVIEW

File: css/components/updates.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  font-size: 14px;
+  line-height: 35px;
+  outline: none;
+}
+
+.updates-form input::placeholder {
+  text-transform: capitalize;
+}
+
+.updates-form input:focus {
```
Comment (Jjurgita) [inline css/components/updates.css:54]: There's no box-shadow in original Neuron page input part.

**Review by InetaVei — APPROVED** [review]
lgtm

**Review by Jjurgita — CHANGES_REQUESTED** [review]
Wrong size h2, size should be same like Creativity or Services sections. Maybe we can create new issue for that and change all h2 at the same time :)

File: css/components/updates.css (hunk 1 above)
Comment (EgleJuske) [reply css/components/updates.css:54]: in the original there is nothing on `:focus` state, which is not good, so I put box-shadow on it to be noticable if you navigate the page only with keyboard.

Comment (Jjurgita) [reply css/components/updates.css:54]: Okey :)

**Review by Jjurgita — APPROVED** [review]

**Review by Ramazinas — APPROVED** [review]

**Pull request merged**